indra/databases/bioregistry.py

```diff
@@ -0,0 +1,129 @@
+"""This module implements a client for using namespace and identifiers
+information from the Bioregistry (bioregistry.io)."""
+import re
+from indra.resources import load_resource_json
+
+# Additional mappings that we need for getting a bioregistry prefix
+# corresponding to an INDRA namespace, not covered by synonyms.
+bioregistry_overrides = {
+    'UP': 'uniprot',
+    'UPPRO': 'uniprot.chain',
+    'UPISO': 'uniprot.isoform',
+    'UPLOC': 'uniprot.location',
+    'REFSEQ_PROT': 'refseq',
+    'PF': 'pfam',
+    'IP': 'interpro',
+    'NONCODE': 'noncodev4.rna',
+    'LNCRNADB': 'rnacentral',
+    'MIRBASEM': 'mirbase.mature',
+    'EGID': 'ncbigene',
+    'NCBI': 'ncbigene',
+    'HGNC_GROUP': 'hgnc.genefamily',
+    'LINCS': 'lincs.smallmolecule',
+    'PUBCHEM': 'pubchem.compound',
+    'CHEMBL': 'chembl.compound',
+    'CTD': 'ctd.chemical',
```

Comment:
ideally this would point to `mesh` instead, but you know best if this is possible